plugins/easyimage/styles/easyimage.css

```diff
@@ -43,6 +43,21 @@ The outline is not a part of the element's dimensions, we have to use a border a
 	max-width: 25%;
 }
 
+.cke_widget_wrapper_easyimage-alignLeft, :not(.cke_widget_wrapper_easyimage):not(.cke_widget_wrapper_easyimage-alignLeft) > .easyimage-alignLeft {
+	float: left;
+	max-width: 25%;
+}
+
+.cke_widget_wrapper_easyimage-alignCenter, :not(.cke_widget_wrapper_easyimage):not(.cke_widget_wrapper_easyimage-alignCenter) > .easyimage-alignCenter {
+	margin: 0 auto;
+	max-width: 90%;
+}
+
+.cke_widget_wrapper_easyimage-alignRight, :not(.cke_widget_wrapper_easyimage):not(.cke_widget_wrapper_easyimage-alignRight) > .easyimage-alignRight {
```

Comment:
I think this particular selector could be joined to the `.cke_widget_wrapper_easyimage-side, :not(.cke_widget_wrapper_easyimage):not(.cke_widget_wrapper_easyimage-side) > .easyimage-side` selector above.